k8s-daemonset/k8s/servicemesh.yml

```diff
@@ -0,0 +1,359 @@
+################################################################################
+# Linkerd Sevice Mesh
+#
+# This is a basic Kubernetes config file to deploy a service mesh of Linkerd
+# instances onto your Kubernetes cluster that is capable of handling HTTP,
+# HTTP/2 and gRPC calls with some reasonable defaults.
+#
+# To configure your applications to use Linkerd for HTTP traffic you can set the
+# `http_proxy` envionment variable to `$(NODE_NAME):4140` where `NODE_NAME` is
+# the name of node on which the application instance is running.  The
+# `NODE_NAME` environment variable can be set with the downward API.
+#
+# If your application does not support the `http_proxy` environment variable or
+# if you want to configure your application to use Linkerd for HTTP/2 or gRPC
+# traffic, you must configure your application to send traffic directly to
+# Linkerd:
+#
+# * $(NODE_NAME):4140 for HTTP
+# * $(NODE_NAME):4240 for HTTP/2
+# * $(NODE_NAME):4340 for gRPC
+#
+# If you are sending HTTP or HTTP/2 traffic directly to Linkerd, you must set
+# the Host/Authority header to `<service>` or `<service>.<namespace>` where
+# `<service>` and `<namespace>` are the names of the service and namespace
+# that you want to proxy to.
+#
+# If your application receives HTTP, HTTP/2, and/or gRPC traffic it must have a
+# Kubernetes Service object with ports named `http`, `h2`, and/or `grpc`
+# respectively.
+#
+# You can deploy this to your Kubernetes cluster by running:
+#   kubectl create ns linkerd
+#   kubectl apply -n linkerd -f servicemesh.yml
+#
+# There are sections of this config that can be uncommented to enable:
+# * CNI compatibility
+# * Automatic retries
+# * Zipkin tracing
+################################################################################
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+  namespace: linkerd
+data:
+  config.yaml: |-
+    admin:
+      port: 9990
+
+    # Namers provide Linkerd with service discovery information.  To use a
+    # namer, you reference it in the dtab by its prefix.  We define 4 namers:
+    # * /io.l5d.k8s gets the address of the target app
+    # * /io.l5d.k8s.http gets the address of the http-incoming Linkerd router on the target app's node
+    # * /io.l5d.k8s.h2 gets the address of the h2-incoming Linkerd router on the target app's node
+    # * /io.l5d.k8s.grpc gets the address of the grpc-incoming Linkerd router on the target app's node
+
+    namers:
```

Comment:
Super minor, but for all of these comments that precede `namers`, `telemetery`, `usage`, and `router`, I'd remove the blank line between the end of the comment and the key to which the comment applies.